src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -3286,13 +3285,66 @@ public bool PackageExistsInPackagesFolder(PackageIdentity packageIdentity)
             return new ResolvedPackage(latestVersion, resolvedPackages.Any(p => p.Exists));
         }
 
+        ///A function that given a list of packages, a version and resolution context returns a list
```

Comment:
What I meant here, is something like this. 
https://github.com/NuGet/NuGet.Client/blob/b14b0500100420e4da687206b4b1cfb08a0a437d/src/NuGet.Core/NuGet.Protocol/Plugins/IPluginManager.cs#L15-L21

Putting the description in a summary element like in the example, shows up in intellisense.

Reply:
//cc @AbdallahAbuHashem 
Change the comment to one like in the above example. 
In VS..do /// and it will autocomplete the params and return types.